src/test/java/net/kaczmarzyk/spring/data/jpa/web/WebRequestProcessingContextPathVariableResolverTest.java

```diff
@@ -45,116 +45,143 @@ public class WebRequestProcessingContextPathVariableResolverTest {
 	@Parameters(name = "TestController: {0}")
 	public static Collection testController() {
 		return Arrays.asList(
-				TestControllerWithClassLevelRequestMappingWithValue.class,
-				TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class,
-				TestControllerWithClassLevelRequestMappingWithPath.class,
-				TestControllerWithClassLevelRequestMappingWithPathAndPathVarWithRegexp.class
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValue.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPath.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPathAndPathVarWithRegexp.class, Boolean.TRUE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValue.class, Boolean.FALSE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithValueAndPathVarWithRegexp.class, Boolean.FALSE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPath.class, Boolean.FALSE },
+				new Object[] { TestControllerWithClassLevelRequestMappingWithPathAndPathVarWithRegexp.class, Boolean.FALSE }
 		);
 	}
-	
+
 	private Class<?> testController;
+	private Boolean getAttributes;
 	
-	public WebRequestProcessingContextPathVariableResolverTest(Class<?> testController) {
+	public WebRequestProcessingContextPathVariableResolverTest(Class<?> testController, Boolean getAttributes) {
```

Comment:
Consider renaming `getAttributes` to: `webRequestAttributesWithExtractedPathVariables`

Reply:
not only consider, but just do it.